R/comp_report.R

```diff
@@ -16,6 +16,7 @@
 #' @param maf_data Optional parameter with maf like df already loaded into R.
 #' @param maf_path Optional parameter with path to external maf like file.
 #' @param this_seq_type Seq type for returned CN segments. One of "genome" (default) or "capture".
+#' @param projection The projection you want back.
```

Comment:
I find this confusing. I think it should say what are the possible values and which one is the default. Maybe it should be a standard way to describe a parameter that repeats among GAMBLR functions.